### PR TITLE
Pull request for (add-files)

### PR DESCRIPTION
**Pull request by ekenecf**
My first module exercise.

1. Added a header to the HTML file 
2. Styled with CSS
3. Used linters to cross-check my file
4. Edited the read-me file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Project Name
```
Comment (CollinsTatang) [inline README.md:3]: - [ ] Kindly replace this with your project name.

File: README.md [hunk 2]
```diff
@@ -16,10 +16,7 @@
 [Live Demo Link](https://livedemo.com)
```
Comment (CollinsTatang) [inline README.md:16]: - [ ] Please remove the live demo link if you don't have a working demo link to put in there.

File: text.md [hunk 3]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (CollinsTatang) [inline text.md:1]: - [ ] Please change the file name to `test.md` and ignored it in the .gitignore file

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @ekenecf , 

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Gitflow flow was correctly used ✔️ 
## Required Changes ♻️
- Kindly add microverse linters to your project  [Microverse linters](https://github.com/microverseinc/linters-config/tree/master/html-css)🤗
- Kindly modified you pull request to have a descriptive title and a short summary describing introduced changes.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @ekenecf, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions:
- Kindly add a link tag to your CSS file in the order your style applied. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**